internal/jimm/admin_test.go

```diff
@@ -38,10 +45,19 @@ func TestLoginDevice(t *testing.T) {
 func TestGetDeviceSessionToken(t *testing.T) {
 	c := qt.New(t)
 	pollingChan := make(chan string, 1)
+	ofgaClient, _, _, err := jimmtest.SetupTestOFGAClient(c.Name())
```

Comment:
Why did this test get an OpenFGA client?

Reply:
jimm constructor needs an openfga client, otherwise it cannot start the role manager